The awk pattern was checked *before* the Perl pattern, so the
perl script with BEGIN{...} would be reported as awk, this patch fixes it.

Upstream-Status: Backport [debian]

Signed-off-by: Christoph Biedl <debian.axhn@manchmal.in-ulm.de>
Signed-off-by: Robert Yang <liezhi.yang@windriver.com>

Rebase on 5.31

Signed-off-by: Fan Xin <fan.xin@jp.fujitsu.com>
---
 magic/Magdir/commands | 1 +
 1 file changed, 1 insertion(+)

diff --git a/magic/Magdir/commands b/magic/Magdir/commands
index 1a46efd..255c04b 100644
--- a/magic/Magdir/commands
+++ b/magic/Magdir/commands
@@ -57,6 +57,7 @@
 0	string/wt	#!\ /usr/bin/awk	awk script text executable
 !:mime	text/x-awk
 0	regex/4096	=^[A-Za-z0-9_]{0,100}BEGIN[A-Za-z0-9_]{0,100}[{]	awk or perl script text
+!:strength - 12
 
 # AT&T Bell Labs' Plan 9 shell
 0	string/wt	#!\ /bin/rc	Plan 9 rc shell script text executable
-- 
1.9.1

